### PR TITLE
Coach page

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]

File: backend/src/models/TeamAndStaffManager.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ class TeamAndStaffManager extends AbstractManager {
 
   findAll() {
     return this.connection.query(
-      `SELECT s.lastname, s.firstname, s.img, t.title, t.division FROM staff AS s LEFT JOIN team AS t ON t.id=staff_id;`
+      `SELECT *
```
Comment (LuigiMenez) [inline backend/src/models/TeamAndStaffManager.js:8]: This could be a nicer indentation.

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Query indentation to be reviewed.

**Review by FrancoisSalmon — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**